### ISSUE TITLE
About the dataset

### ISSUE THREAD

**Issue by Breezewrf** [bcmi/SLBR-Visible-Watermark-Removal#5]
it seems that the dataset CLWD only includes one sub folder, however, the code to train need "train" and "test" folders, could you give me some suggestion on how to split the train set and test set? Thanks a lot !

**Comment by Breezewrf** [comment]
got it 
I'll appreciate it if you release the pretrained model as soon as possible sincerely
